demo-terraform/aws/serverless-app/cognito.tf

```diff
@@ -1,30 +1,84 @@
 
 locals {
-    cognito_user_pool_name = "user-pool-${random_string.uniq.result}"
+  cognito_user_pool_name = "user-pool-${random_string.uniq.result}"
 }
 
 ## Creates User Pool
 resource "aws_cognito_user_pool" "pool" {
```

Comment:
**Accurics remediation**
 
**[MEDIUM]: Ensure standard password policy must be followed with password having at least 1 symbol**
 
**Vulnerability**
Weak Password without a symbol can be easily cracked using brute force.
 
**Remediation**
Password policy for AWS accounts can be created and managed in AWS Console.
 In AWS Console - 
 1. Sign in to the AWS Console and go to the IAM console.
 2. Choose Account settings in the navigation pane.
 3. Select 'Change password policy' in the Password policy section.
 4. Select the password policy configuration.
 5. Select Save changes.

source: accurics-